README.md

```diff
@@ -1,14 +1,260 @@
+a compliance-management framework built on Puppet.
+
+If you find any issues, they can be submitted to our
+[JIRA](https://simp-project.atlassian.net/).
+
+Please read our [Contribution Guide](https://simp-project.atlassian.net/wiki/display/SD/Contributing+to+SIMP)
+and visit our [developer wiki](https://simp-project.atlassian.net/wiki/display/SD/SIMP+Development+Home).
+
+This module is optimally designed for use within a larger SIMP ecosystem, but it
+can be used independently:
+* When included within the SIMP ecosystem, security compliance settings will be
+managed from the Puppet server.
+* In the future, all SIMP-managed security subsystems will be disabled by
+default and must be explicitly opted into by administrators.  Please review
+*simp/simp_options* for details.
+
+## Module Description
+
+Svckill is a system that attempts to run with the security best practice that
+"No unnecessary services should be running on the system."
+
+The way svckill works is to fetch all services on the running system and then
+shutdown and disable any that are not declared in a Puppet manifest (or ignore
+list/file) somewhere.
+
+## Setup
+
+### What simp svckill affects
+
+*simp/svckill* effects ALL services on a given node. If this class is included
+on a node, all services not declared in a puppet service resource or a svckill
+ignore list will be disabled and turned off on a system. The following are
+exceptions:
+
+```
+    The following services are hard coded to never be killed by svckill:
+      * amtu
+      * blk-availability
+      * crond
+      * ebtables
+      * gpm
+      * haldaemon
+      * ip6tables
+      * iptables
+      * irqbalance
+      * killall
+      * libvirt-guests
+      * lvm2-monitor
+      * mcstrans
+      * mdmonitor
+      * messagebus
+      * netcf-transaction
+      * netfs
+      * netlabel
+      * network
+      * ntpdate
+      * portreserve
+      * puppet
+      * restorecond
+      * sandbox
+      * sshd
+      * sysstat
+      * udev-post
+      * getty*
+      * dbus*
+
+   These are here because their status function is broken
+      * krb524
+      * mdmpd
+      * readahead_later
+      * rawdevices
+      * lm_sensors
+      * kudzu
+```
+
+### Setup Requirements
+
+### Beginning with svckill
+
+You can set up svckill on a node by:
+
+```puppet
+include 'svckill'
+```
+
+## Usage
+
+### I have a service I don't want puppet to kill on a single node
+
+There are two ways to solve this problem:
+
+Declare the service in puppet:
+
+```puppet
+service { 'myservice':
+  ensure => running,
+}
+```
+
+or Declare the service in an ignore list in svckill:
+
+```puppet
+svckill { 'myservice':
+  ignore => ['myservice'],
+}
+```
+
+### I want to ignore a list of services I deploy in a file
+
+```puppet
+svckill { 'ourservices':
+  ignorefiles => '/opt/services',
+}
+```
+
+### I want to set a list of services allowed in my entire infrascture
```

Comment:
I think you should have a section setting svckill::ignore in hiera. It might be easier for some people.

Also, infrastructure.